src/view/screens/Search/Search.tsx

```diff
@@ -172,6 +227,7 @@ function SearchScreenSuggestedFollows() {
       contentContainerStyle={{paddingBottom: 1200}}
```

Comment:
Need to remove this padding (or rather probably set it to like 200px or so), otherwise  you scroll down way too far.

Reply:
yeah didn't want to touch it cause not sure why it's there

Reply:
Looks like it is there because we used to just show the keyboard overtop of the suggestions. now though (i think? it does for me anyway) we show a blank screen or list previous searches (only up to five) so we don't need to be able to scroll below the keyboard.